### DISCUSSION TITLE
How to prefix media included in CSS

### DISCUSSION THREAD

**Discussion by simPod** [vercel/next.js#19863] [Help]
I have a `background-image: url()` in my CSS. It gets processed into something like `url(/_next/static/images/data-ca5b4b33bb115bdb9a3e699262772271.jpg)` so it goes through next.js.

How to prefix it? My site is running at `localhost:3000/app` so `url(/_next/static/images/data-ca5b4b33bb115bdb9a3e699262772271.jpg)` gives 404.

`/app/_next/static/images/data-ca5b4b33bb115bdb9a3e699262772271.jpg` works. Tried setting `basePath` and `assetPrefix` but it has no effect.

Next.js v10.0.3

**Comment by eweseong** [comment]
Hi is there a solution to this? I'm having similar troubles of loading locally served fonts in CSS with a custom `basePath`. I'm appending the custom base path in the CSS files as a workaround, but would love to know if there's a setting which we could configured in `next.config.js`
